x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/feature_table.test.tsx

```diff
@@ -28,9 +28,12 @@ interface TestConfig {
   role: Role;
   privilegeIndex: number;
   calculateDisplayedPrivileges: boolean;
+  canCustomizeSubFeaturePrivileges: boolean;
```

Comment:
The diff doesn't show it, but most of these tests are the same, with the distinction that they've been updated to run with both sub-features enabled and disabled.